### DISCUSSION TITLE
Is there’s a way to have the API on a domain and the admin panel on a different domain

### DISCUSSION THREAD

**Discussion by egahmad** [directus/directus#8682] [Archive]
I've gone through the configuration and the env documentation looking for an answer with no luck.

Even if theres a way to make the API respond on a port and the admin panel on a different port in docker? 

Any idea?

**Comment by joselcvarela** [comment]
Hello @egahmad 
One way you could achieve what you want is to run two instances of Directus image. The API instance should set this environment variable `SERVE_APP=false`. This tells the container to not serve requests on `/admin` endpoint. The APP instance could run normally. 

Though, you will need to connect both instances to the same database which by default each instance creates a new database for itself. To get around this, you should set the `DB_` environment variables. Check here an example with postgres: https://docs.directus.io/guides/installation/docker/#docker-compose

**Reply by endlessuniverse** [reply]
Hello @joselcvarela ! Can we host a frontend app at the root of the same domain that the Directus image with the /admin endpoint?
Thank you!
